sdk/react/panels/videoView.js

```diff
@@ -78,22 +78,14 @@ class VideoView extends Component {
     if (this.props.live) {
       const isLive = this.props.playhead >= this.props.duration * VALUES.LIVE_THRESHOLD;
       return ({
-        label:
-          isLive ? Utils.localizedString(this.props.locale, 'LIVE', this.props.localizableStrings) :
-            Utils.localizedString(this.props.locale, 'GO LIVE', this.props.localizableStrings),
-        onGoLive: isLive ? null : this.onGoLive
+        label: Utils.localizedString(this.props.locale, 'LIVE', this.props.localizableStrings),
```

Comment:
Could you check this line with linter? I see that we can destruct props.

Reply:
Done